wp-content/themes/theme_name/inc/enqueue.php

```diff
@@ -8,26 +8,16 @@
 if ( ! function_exists( 'register_scripts' ) ) {
   function register_scripts() {
     // jQuery
-    // wp_deregister_script( 'jquery-migrate' );
     wp_deregister_script( 'jquery' );
+    wp_register_script( 'jquery', "https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js", array(), '3.1.1' );
+    wp_enqueue_script('jquery');
 
-    $cdn_jQuery = wp_register_script( 'jquery-cdn', 'https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js' );
-
-    if ( $cdn_jQuery ) {
-      wp_enqueue_script( 'jquery-cdn' );
-    } else {
-      wp_register_script( 'jquery' );
-      wp_enqueue_script( 'jquery' );
-    }
+    wp_deregister_script( 'jquery-migrate' );
```

Comment:
These lines shouldn't have been removed, we need a fallback in case a cdn fails.